c2corg_api/views/document_associations.py

```diff
@@ -12,23 +13,25 @@
 from c2corg_api.views.document_schemas import waypoint_documents_config, \
     route_documents_config, user_profile_documents_config, \
     image_documents_config, article_documents_config, area_documents_config, \
-    outing_documents_config
+    outing_documents_config, report_documents_config
 from c2corg_api.views.validation import updatable_associations
 from sqlalchemy.sql.expression import or_, and_
 
 associations_to_include = {
     WAYPOINT_TYPE: {
         'waypoints', 'waypoint_children', 'routes', 'images', 'articles'},
     ROUTE_TYPE:
-        {'waypoints', 'routes', 'images', 'articles'},
+        {'waypoints', 'routes', 'images', 'articles', 'reports'},
```

Comment:
In v5, the reports are not included for routes, outings, articles or images. Right, @asaunier?